python/paddle/tensor/manipulation.py

```diff
@@ -121,6 +121,37 @@ def concat(x, axis=0, name=None):
     return paddle.fluid.layers.concat(input=x, axis=axis, name=name)
 
 
+def broadcast_tensors(x, name=None):
+    """
+
+    This OP broadcast a list of tensors following broadcast semantics
```

Comment:
Add API Doc in [https://github.com/PaddlePaddle/docs](FluidDocRepo)

Reply:
Screenshot added.